### ISSUE TITLE
NSP Security Check 534: failed because of debug module

### ISSUE THREAD

**Issue by DidierHoarau** [Kikobeats/hyperdiff#7]
Hi,

In one of our projects we use hyperdiff. (cool module by the way, thanks!)

We got a security exception because of Hyperdiff > Debug:

```
$ nsp check
(+) 1 vulnerabilities found
┌───────────────┬────────────────────────────────────────────────────────────────────────────────────────┐
│               │ Regular Expression Denial of Service                                                   │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ Name          │ debug                                                                                  │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ CVSS          │ 3.7 (Low)                                                                              │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ Installed     │ 3.0.1                                                                                  │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ Vulnerable    │ <= 2.6.8 || >= 3.0.0 <= 3.0.1                                                          │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ Patched       │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                                           │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ Path          │ tg-nms-acct@0.1.0 > hyperdiff@2.0.3 > debug@3.0.1                                      │
├───────────────┼────────────────────────────────────────────────────────────────────────────────────────┤
│ More Info     │ https://nodesecurity.io/advisories/534                                                 │
└───────────────┴────────────────────────────────────────────────────────────────────────────────────────┘

```

Based on what I see...
- Hyperdiff already made the update to debug 3.1.0: https://github.com/Kikobeats/hyperdiff/commit/de5a00ace7fdc6439b96eb3b6388c855ec3f183b
- But after the fix no new release of Hyperdiff has been made.

Would the solution be "simply" to issue a new release?
If so, can someone please release a new version?

Thank you in advance,

**Comment by Kikobeats** [comment]
Hey,

Thanks for report that. I released a new patch version (`2.0.4`)

Can you check it?

**Comment by DidierHoarau** [comment]
Hi,

I've checked it just now and no more NSP check error with v 2.0.4. 
Thank a lot!

Regards,
